client/components/admin/AdminDashboard.js

```diff
@@ -8,6 +8,7 @@ const AdminDashboard = ({ isAdminLoggedIn }) => {
 
   return (
     <div>
+      
       <h2>Admin Dashboard</h2>
       <ul>
       {isAdminLoggedIn && (
```

Comment:
We can remove this link to '/' on line 25 now, when I click that I get a weird button overlay happening.
<img width="1200" alt="Screen Shot 2023-04-06 at 3 38 58 PM" src="https://user-images.githubusercontent.com/71556587/230490673-d61dbe4a-17f7-4d7f-b847-2c7ff15d58c5.png">

Reply:
Thanks for catching that! We will definitely make this fix.